app/tests/src/test_ingest_policy_pdfs.py

```diff
@@ -39,8 +35,18 @@ def test__ingest_policy_pdfs(
 
     with caplog.at_level(logging.INFO):
         if file_location == "local":
-            _ingest_policy_pdfs(db_session, policy_local_file, doc_attribs)
+            _ingest_policy_pdfs(policy_local_file, doc_attribs)
```

Comment:
Could be simplified to:
```suggestion
            _ingest_policy_pdfs("/app/tests/docs/", doc_attribs)
```